src/app/shell/info/rules/rules.component.ts

```diff
@@ -26,6 +29,7 @@ export class RulesComponent {
         this.navigationBarService.createOneNavPath({ name: NavBarName.LawsAndRegulations, isActive: false, disable: true })
```

Comment:
this.store.dispatch([this.navigationBarService.createOneNavPath({ name: NavBarName.LawsAndRegulations, isActive: false, disable: true }),new GetLawsAndRegulations())])

Reply:
VSC displays as incorrect syntax.
Left it as in other components as About and Support